make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -295,7 +295,7 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBLCMS, \
         libawt/java2d, \
     HEADERS_FROM_SRC := $(LIBLCMS_HEADERS_FROM_SRC), \
     DISABLED_WARNINGS_gcc := format-nonliteral type-limits \
-        misleading-indentation undef unused-function stringop-truncation, \
+        undef unused-function stringop-truncation, \
```

Comment:
Ok, let's not touch this global warning and rename the issue to "Fix misleading-indentation warnings in core JDK libraries". Leave the Awt2d change to another day.

Reply:
What is the problem with Awt2d? Other conflicts?

Reply:
I am not completely sure all `misleading-indentation` warnings are fixed in Awt2d, this would require some more testing. I'd defer that to #10453.

Reply:
Awt2dLibraries.gmk in https://github.com/openjdk/jdk/pull/10453 seems more about clang if I'm not mistaken, while this change is about gcc.
The JDK successfully builds on several platforms with this change.

What else could go wrong? I'm asking because I'm not a build engineering expert at all.

Reply:
This could fail the build on some toolchains. Cleanliness and least surprise: this PR changes the particular files, and so it should remove the warnings for those specific files.